app/controllers/users_controller.rb

```diff
@@ -1,4 +1,29 @@
 class UsersController < ApplicationController
   def new
+    @user = User.new
   end
+
+  def show
+    @user = User.find_by id: params[:id]
+    return if @user.present?
+    flash[:success] = t :show 
```

Comment:
Em, please note that, this line will be executed when the @user is nil. If the @user is present, then the this line will not execute and request will return from previous line. 
So, if the @user is nil/not found, should we show success message?

Reply:
oops... sorry, I'm so lazy, I coppyed the other line

Reply:
without fix